cookbook/cookbook.md

```diff
@@ -2613,6 +2613,187 @@ Flow for Creating Guarded Relayed Transactions:
 4. Then, the guardian signs.
 5. Finally, the relayer signs before broadcasting.
 
+### Multisig
+
+The sdk contains components to interact with the (Multisig Contract)[https://github.com/multiversx/mx-contracts-rs/releases/tag/v0.45.5].
+We can deploy a multisig smart contract, add members, propose and execute actions and query the contract.
+The same as the other components, to interact with a multisig smart contract we can use either the MultisigController or the MultisigTransactionsFactory.
+
+These operations can be performed using both the **controller** and the **factory**. For a complete list of supported methods, please refer to the autogenerated documentation:
+- [`MultisigController`](https://multiversx.github.io/mx-sdk-js-core/v14/classes/MultisigController.html)
+- [`MultisigTransactionsFactory`](https://multiversx.github.io/mx-sdk-js-core/v14/classes/MultisigTransactionsFactory.html)
+
+#### Deploying a Multisig Smart Contract using the controller
+```js
+{
+    const abi = await loadAbiRegistry("src/testdata/multisig-full.abi.json");
+    const bytecode = await loadContractCode("src/testdata/multisig-full.wasm");
```

Comment:
now that I see it, I am not sure we should use these functions since they are our testutils.